contracts/Market.sol

```diff
@@ -749,21 +755,24 @@ contract Market is
     view
     returns (uint256)
   {
-    return amount + this.calculateNoriFee({amount: amount});
+    return
+      (amount * _priceMultiple) / 100 + this.calculateNoriFee({amount: amount});
```

Comment:
```suggestion
      ((amount * _priceMultiple) / 100) + this.calculateNoriFee({amount: amount});
```
Although not needed, I find the above notation easier to read

Reply:
moot after using muldiv